rust-toolchain.toml

```diff
@@ -1,3 +1,3 @@
 [toolchain]
-channel = "1.66"
```

Comment:
(non-blocking) might want to update clippy target to 1.71 as well to get new lints.